src/backend/table_builder.rs

```diff
@@ -101,6 +101,7 @@ pub trait TableBuilder: IndexBuilder + ForeignKeyBuilder + QuotedBuilder + Table
                 TableOpt::Engine(s) => format!("ENGINE={}", s),
                 TableOpt::Collate(s) => format!("COLLATE={}", s),
                 TableOpt::CharacterSet(s) => format!("DEFAULT CHARSET={}", s),
+                TableOpt::Strict => "STRICT".to_string(),
```

Comment:
Hey @evgeniy-terekhin, sorry for the delay.

Since `STRICT` is a SQLite only keyword, we better append it on SQLite's table_builder instead of doing it in the common table_builder.